pkg/apis/servicecatalog/validation/instance_test.go

```diff
@@ -976,11 +976,11 @@ func TestValidatePlanReference(t *testing.T) {
 		{
 			name: "invalid -- external name, k8s plan",
 			ref: servicecatalog.PlanReference{
-				ExternalClusterServiceClassName: externalClusterServiceClassName,
-				ClusterServicePlanName:          externalClusterServicePlanName,
+				ClusterServiceClassClientName: clusterServiceClassClientName,
+				ClusterServicePlanName:        clusterServicePlanClientName,
```

Comment:
shouldn't this field in the `PlanReference` type have "ClientName" in its name as well?

Reply:
I think `ClusterServicePlanName` refers to the k8s name. In that resource we now have both, `ClusterServicePlanClientName` and `ClusterServicePlanName`.  I'm pretty sure it wouldn't be very kube-ish but making it `ClusterServicePlanK8sName` would clarify it.

Reply:
yeah I got confused.

Reply:
adding extra levels with grouping k8s- and client- names together would help, but I know you don't like extra nesting in YAML/JSON :)